### PR TITLE
Docs: Update azuread.md to reflect Azure Portal Interface

### PR DESCRIPTION
**Pull request by trallnag**
Few months ago there was an update to the **App Registrations** page in Azure Portal (https://docs.microsoft.com/de-de/azure/active-directory/develop/app-registrations-training-guide-for-app-registrations-legacy-users). This updates the doc to fit the few minor changes:

* Sign-on url renamed to redirect url
* Endpoints overview moved
* Adding users/groups moved

### CODE REVIEW

**Comment by oddlittlebird** [comment]
No complaints from a docs standpoint, @daniellee , please have someone review the content.

@trallnag , please resolve the conflicts.

**Comment by oddlittlebird** [comment]
Go away, Stalebot!

File: docs/sources/auth/azuread.md [hunk 1]
```diff
@@ -22,18 +22,17 @@ To enable the Azure AD OAuth2 you must register your application with Azure AD.
 
 1. Log in to [Azure Portal](https://portal.azure.com) and click **Azure Active Directory** in the side menu.
 
-1. Click **App Registrations** and add a new application registration:
-   - Name: Grafana
-   - Application type: Web app / API
-   - Sign-on URL: `https://<grafana domain>/login/azuread`
+1. Click **App Registrations** and add a new application registration with a fitting name.
+   
+1. Click on **Authentication** then **Add a platform** and pick **Web App**.
```
Comment (aknuds1) [inline docs/sources/auth/azuread.md:27]:
```suggestion
1. Click on **Authentication**, then **Add a platform** and pick **Web App**.
```

**Review by aknuds1 — CHANGES_REQUESTED** [review]
Just a nit.

**Comment by aknuds1** [comment]
We need someone with Azure familiarity to review. @daniellee?

**Comment by oddlittlebird** [comment]
@kylebrandt ?

**Comment by kylebrandt** [comment]
note: english documentation link is https://docs.microsoft.com/en-us/azure/active-directory/develop/app-registrations-training-guide-for-app-registrations-legacy-users

**Review by kylebrandt — APPROVED** [review]

**Pull request merged**